packages/typespec-ts/test/integration/routes.spec.ts

```diff
@@ -32,13 +32,94 @@ describe("RoutesClient Rest Client", () => {
     assert.strictEqual(result.status, "204");
   });
 
-  it.skip("should have PathParameters SimpleExpansion Standard primitive", async () => {
+  it("should have allowReserved: true", async () => {
     const result = await client
-      .path("/routes/path/simple/standard/primitive{param}", "a")
+      .path("/routes/path/reserved-expansion/template/{param}", {
+        value: "foo/bar baz",
+        allowReserved: true
+      })
       .get();
     assert.strictEqual(result.status, "204");
   });
 
+  it("should have explode: true array", async () => {
+    const result = await client
+      .path("/routes/query/query-expansion/explode/array")
+      .get({
+        queryParameters: {
+          param: {
+            value: ["a", "b"],
+            explode: true,
+            style: "form"
+          }
+        }
+      });
+    assert.strictEqual(result.status, "204");
+  });
+
+  it("should have explode: true record", async () => {
+    const result = await client
+      .path("/routes/query/query-expansion/explode/record")
+      .get({
+        queryParameters: {
+          param: {
+            value: { a: 1, b: 2 },
+            explode: true,
+            style: "form"
+          }
+        }
+      });
+    assert.strictEqual(result.status, "204");
+  });
+
+  it("should have explode: true primitive", async () => {
```

Comment:
This case failed with same reason, I plan to not generate wrapper type for primitive because there explode or style information will have no effect for the url. But @timovv I think you could decide in core side if we need to support primitive value type.

![image](https://github.com/user-attachments/assets/d165a5fa-5f25-41e1-aa91-fbeeb71d2e59)

Reply:
We _can_ not throw here (in that case we would just ignore `explode` if we don't get an array), but I agree with your plan and think it would be better if we just didn't generate the wrapper in this case, since the combination of primitive and `explode: true` doesn't really make sense.